contrib/openshift/build_and_deploy.sh

```diff
@@ -3,17 +3,15 @@ set -exuo pipefail
 
 : "${QUAY_USER:?Missing QUAY_USER variable.}"
 : "${QUAY_TOKEN:?Missing QUAY_TOKEN variable.}"
-: "${REPOSITORY:=quay.io/app-sre}"
+: "${REGISTRY:=quay.io}"
+: "${REPOSITORY:=${REGISTRY}/app-sre}"
 : "${IMAGE:=${REPOSITORY}/clair}"
 GIT_HASH="$(git rev-parse --short=7 HEAD)"
 
 git archive HEAD |
 	docker build -t clair-service:latest -
```

Comment:
Is there a good reason to keep using docker for the build instead of podman? I think when the first version of this script was written podman wasn't really usable in the runner.

Reply:
That is a good point, I'll update that